Source/ACE.Database/ShardDatabase.cs

```diff
@@ -156,6 +164,21 @@ protected override void InitializePreparedStatements()
             ConstructStatement(ShardPreparedStatement.InsertTextureOverridesByObject, typeof(TextureMapOverride), ConstructedStatementType.InsertList);
             ConstructStatement(ShardPreparedStatement.InsertPaletteOverridesByObject, typeof(PaletteOverride), ConstructedStatementType.InsertList);
             ConstructStatement(ShardPreparedStatement.InsertAnimationOverridesByObject, typeof(AnimationOverride), ConstructedStatementType.InsertList);
+
+            // FIXME(ddevec): Use max/min values defined in factory -- this is just for demonstration purposes
```

Comment:
I really dislike this FIXME.  I guess we can keep it for now and make a trello card about actually doing Guid manager allocation properly